core/src/main/resources/jenkins/security/ResourceDomainConfiguration/config.jelly

```diff
@@ -25,7 +25,7 @@ THE SOFTWARE.
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
     <f:section title="${%Serve resource files from another domain}">
-        <f:entry title="${%Resource root URL}" field="url">
+        <f:entry title="${%Resource Root URL}" field="url">
```

Comment:
Note that it will break external localizations like Chinese Localization plugin. Heads-up to @LinuxSuRen

Reply:
How about just improve it in the English properties?

Reply:
@LinuxSuRen: No.

Would it help if I made a PR to update your thing?

Reply:
@LinuxSuRen: Imagine we misspelled "Chinese" as `_("Cihnese")` and that was the localization key. You could say "Why don't you just fix the English properties file to say `Chinese`?", but that would be wrong. And I'd be shocked if you actually suggested this.

What I'm doing here is effectively fixing the canonical spelling of terms. In part to prevent future confusion. One doesn't do that by wallpapering over it in a localization.

In a past life, I maintained a complete retranslation of a product into fixed English. At a certain point, the product management acknowledged that I was better than they were and agreed to try to accept my input.

If you need help updating your localization, I'm happy to make the changes on your side.

Beyond that, I am not remotely interested in stop energy here.